config.toml

```diff
@@ -46,4 +46,6 @@ publish_hold = [
     "/topics/internals-vm/",
     "/topics/internals/",
     "/topics/rdd/"
-]
\ No newline at end of file
+]
+
+banner = "The Release Candidate for Valkey 8.1 is available! [Check it out on GitHub](https://github.com/valkey-io/valkey/releases/tag/8.1.0-rc1)"
```

Comment:
```suggestion
banner = "The Release Candidate for Valkey 8.1 is available! [Check it out!](https://valkey.io/download/releases/v8-1-0/)"
```
This includes links to the containers as well as the github release, might be a bit more generic?